CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 7.1.0 (IN PROGRESS)
 
 * Display the “Record deleted” label in version history only if the UUID no longer exists. Refs UIOR-1355.
+* Fix the actions dropdown in PO and PO Line details before redirecting to the receiving app. Refs UIREC-424.
```

Comment:
In the JIRA ticket, the corresponding project where the changes are occurring must be specified—in this case, `ui-orders` (`UIOR`). The branch name should also reflect this project.  

Please update the project in the JIRA ticket to `ui-orders` and ensure the correct ticket number is specified in the changelog.  

I don’t think it’s necessary to redo the PR and rename the branch since it will be deleted after the merge. However, the PR title and changelog should contain the correct ticket number.  

The title should follow this format:   **[JIRA ticket number] Description**  
For example:  `UIOR-777 Close actions dropdown before navigation to another page`